src/app/shared/modules/switch-language/switch-language.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { NgModule } from '@angular/core';
+import { TestBed, waitForAsync } from '@angular/core/testing';
+import { ModalConfirmComponent } from '@keira-shared/modules/modal-confirm/modal-confirm.component';
+import { TranslateTestingModule } from '@keira-shared/testing/translate-module';
+import { TranslateService } from '@ngx-translate/core';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { SwitchLanguageService } from './switch-language.service';
+
+@NgModule({
+  declarations: [ModalConfirmComponent],
+  imports: [],
+})
+class TestModule {}
+
+describe('SwitchLanguageService', () => {
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [ModalModule.forRoot(), TestModule, TranslateTestingModule],
```

Comment:
why are we creating and importing `TestModule` to declare `ModalConfirmComponent` while we could just add it inside the `declarations: [ .... ]` array here?

Reply:
removed, it was not used